tox.ini

```diff
@@ -17,6 +18,7 @@ deps =
     16: Django >= 1.6, < 1.7
     17: Django >= 1.7, < 1.8
     18: Django >= 1.8, < 1.9
+    19: https://github.com/django/django/tarball/master
```

Comment:
We should add the Django 1.9 tests to the `.travis.yml` file as well to make sure that the tests also run in CI.